ajax/libs/formulas.js/package.json

```diff
@@ -0,0 +1,19 @@
+{
+    "author": "Ismael Ghalimi <ismael@stoic.com>",
+    "name": "formula.js",
+    "version": "1.0.8",
+    "description": "JavaScript implementation of most Microsoft Excel formula functions",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/sutoiku/formula.js.git"
+    },
+    "autoupdate": {
```

Comment:
@nicolashefti We prefer to use npm auto-update config if the library had been published on npm. Thanks. ref: [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md)